src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -332,4 +332,62 @@ private boolean isMatchingByLineAndColumnAndTokenType(DetailAST ast) {
                 && expandedTabColumn(ast) == columnNumber
                 && (tokenType == 0 || tokenType == ast.getType());
     }
+
+    /**
+     * Controls true escape characters.
+     *
+     * @param chr character from parent method encode() .
+     * @param string StringBuilder from parent method encode().
+     */
+    private static void fixEscape(char chr, StringBuilder string) {
```

Comment:
Please suggest I fine method name that passes the Questionable name string check of team city. I tried a bunch of names but neither of them get past it.

Reply:
correctEscapedChar()